docs/walrus-sites/tutorial-install.md

```diff
@@ -44,3 +60,52 @@ Commands:
 
   ⋮
 ```
+
+## Configuration
+
+The `site-builder` tool needs a configuration file to work.
+This file is called `sites-config.yaml` and looks like this:
+
+```yaml
+# module: site
+# portal: walrus.site
+package: 0xc5bebae319fc9d2a9dc858b7484cdbd6ef219decf4662dc81a11dc69bb7a5fa7
+# general:
+#   rpc_url: https://fullnode.testnet.sui.io:443
+#   wallet: /path/to/.sui/sui_config/client.yaml
+#   walrus_binary: /path/to/walrus
+#   walrus_config: /path/to/devnet_deployment/client_config.yaml
+#   gas_budget: 500000000
+```
+
+As you can see, the configuration file is quite simple.
+The only mandatory field is the `package` field,
+which represents the Sui object ID of the Walrus Sites
+smart contract. You can find the latest version of the
+package in the [Walrus Sites repository](https://github.com/MystenLabs/walrus-sites/tree/testnet)
+on the `testnet` branch.
+
+```admonish danger title="Walrus Sites stable branch"
+The stable branch of Walrus Sites is `testnet`. Make sure that you always pull the latest changes from there.
+```
+
+You can define the location of the `sites-config.yaml` file using the `--config`
+flag when running the `site-builder` commands like so:
+
+```sh
+site-builder publish <build-directory-of-a-site> --config /path/to/sites-config.yaml
+```
+
+But if are not a fan of repeating the same flags over and over,
+it's always easier to have the configuration file in
+one of the [default locations](../usage/setup.html#config-custom-path).
```

Comment:
Suggestion: these are the default locations for walrus.
maybe explicitly say here which are the locations for sites